encryption-at-rest.md

```diff
@@ -65,9 +65,31 @@ TiKV 当前支持的加密算法包括 AES128-CTR、AES192-CTR、AES256-CTR 和
 
 无论用户配置了哪种数据加密方法，数据密钥都使用 AES256-GCM 算法进行加密，以方便对主密钥进行验证。所以当使用文件而不是 KMS 方式指定主密钥时，主密钥必须为 256 位（32 字节）。
 
```

Comment:
Preview of this PR: https://pingcap-docsite-preview-git-pr-qiancai.vercel.app/zh/tidb/dev/encryption-at-rest